src/wpimath/Geometry/Pose2d.cs

```diff
@@ -75,7 +75,7 @@ public partial class Pose2dJsonContext : JsonSerializerContext
     public Translation2d Translation { get; init; }
     [JsonInclude]
     [JsonPropertyName("rotation")]
-    public Rotation2d Rotation { get; init; }
+    public Rotation2d Rotation { get; init; } = new Rotation2d();
```

Comment:
This is going to be flagged now. I made it so field initializers are disallowed in WPIMath, and instead you should create an explicit parameterless constructor that calls one of the other constructors.